stdlib/buffer.gr

```diff
@@ -120,7 +116,6 @@ let addInt32help = (value, buffer) => {
 /**
  * @section Values: Functions for working with the Buffer data type.
  */
-
```

Comment:
Would be good to preserve these

Reply:
Yup, that's the problem I spotted, I can fix

Reply:
These are now preserved